source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx

```diff
@@ -46,28 +46,15 @@ export type IProps =
   & i18nProps
   & IDataActionsProps;
 
-const mapRecordsToProps = (ownProps: PassedProps) => {
-  const { match } = ownProps;
-  const { params: { orgId } } = match;
-
-  return {
-    cacheKey: [`org-${orgId}`],
-    organization: [q => buildFindRecord(q, TYPE_NAME, orgId), buildOptions({
-      include: ['organization-product-definitions.product-definition','groups']
-    })],
-  };
-};
-
 class SettingsRoute extends React.Component<IProps> {
 
-  updateOrganization = async (payload: OrganizationAttributes) => {
+  updateOrganization = (payload: OrganizationAttributes) => {
 
     const { t, updateAttributes } = this.props;
 
     try {
-      await updateAttributes(payload);
+      updateAttributes(payload);
```

Comment:
why was this await removed?

Reply:
I added them back, not sure why is not showing it